### PR TITLE
Yura maliutin

### PR DESCRIPTION
**Pull request by YuraM8**

### CODE REVIEW

**Comment by YuraM8** [comment]
i add HW 6 - 8

File: Yura_M/HW6/Task_1.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#     elif n % 2 != 0 and n % 3 != 0:
+#         not_divisible_2_3.append(n)
+
+
+# print("even numbers: ", parny)
+# print("odd numbers: ", neparny)
+# print("not divisible by 2 and 3: ", not_divisible_2_3)
+
+
+############################# or ###########################
```
Comment (kolyasalubov) [inline Yura_M/HW6/Task_1.py:25]: good job

File: Yura_M/HW6/Task_2.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Task2. Write a script that checks the login that the user enters.
```
Comment (kolyasalubov) [inline Yura_M/HW6/Task_2.py:1]: good job

File: Yura_M/HW7/CodRaner.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# 1
```
Comment (kolyasalubov) [inline Yura_M/HW7/CodRaner.py:1]: good job

File: Yura_M/HW7/Task_1.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Task1. Write a function that returns the largest number of two
+# numbers
+# (use DocStrings documentation strings in the function).
+
```
Comment (kolyasalubov) [inline Yura_M/HW7/Task_1.py:4]: good job

File: Yura_M/HW7/Task_2.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Task2. Write a program that calculates the area of a rectangle,
+# triangle and circle
+# (write three functions to calculate the area, and call them in the
+# main program depending on the user's choice).
+
```
Comment (kolyasalubov) [inline Yura_M/HW7/Task_2.py:5]: good job

File: Yura_M/HW7/Task_3.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+
+# word = string_f(str(input("enter your words: ")))
+
+# print(word)
+
+
+################################################################## HW 7 Task 3##############
+
+
```
Comment (kolyasalubov) [inline Yura_M/HW7/Task_3.py:28]: good job

File: Yura_M/HW8/Task 1/main.py [hunk 7]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (kolyasalubov) [inline Yura_M/HW8/Task 1/main.py:1]: good job

File: Yura_M/HW8/Task 3/Find_area.py [hunk 8]
```diff
@@ -0,0 +1,27 @@
+from math import pi
```
Comment (kolyasalubov) [inline Yura_M/HW8/Task 3/Find_area.py:1]: good job

File: Yura_M/HW8/Task 3/choice.py [hunk 9]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (kolyasalubov) [inline Yura_M/HW8/Task 3/choice.py:1]: good job

File: Yura_M/HW8/Task_2.py [hunk 10]
```diff
@@ -0,0 +1,20 @@
+import re
```
Comment (kolyasalubov) [inline Yura_M/HW8/Task_2.py:1]: good job
look at this implementation

import re
from getpass import getpass

while True:
    password = getpass("Enter your password: ")
    if not password.strip():
        print("Error: Password cannot be blank.")
        continue
    if len(password) < 6 or len(password) > 16:
        print("Error: Password length must be between 6 and 16 characters.")
        continue
    if not re.search(r"[a-z]", password) or not re.search(r"[A-Z]", password) or not re.search(r"\d", password):
        print("Error: Password must contain at least one lowercase letter, one uppercase letter, and one digit.")
        continue
    if not re.search(r"[$#@]", password):
        print("Error: Password must contain at least one of the following characters: $ # @")
        continue
    print("Congratulations, your password is valid!")
    break

**Pull request merged**